### PR TITLE
Add option to allow package upgrades.

### PR DESCRIPTION
**Pull request by fjgarlin**
Screenshot 2022-06-08 at 17 36 02

Running the composer commands on the lastest Drupal version fails for `drupal/core-dev`. Adding the `-W` fixes the issue.

### CODE REVIEW

**Comment by fjgarlin** [comment]
`-W` requires composer 2. Checking options.

**Comment by fjgarlin** [comment]
@juampynr - as seen on the screenshot, it was failing for the latest D9 version. 

I added the `-W` option to the composer command, which also needs composer 2 (the image seems to be using composer 1).

File: setup-bitbucket.sh
Comment (juampynr) [inline setup-bitbucket.sh:32]: LOL, I had no idea what -W was. I think that I used it before but it was the long option name.

**Review by juampynr — APPROVED** [review]
Approving since this looks good so far.

File: dist/bitbucket/RoboFile.php
Comment (fjgarlin) [inline dist/bitbucket/RoboFile.php:167]: `coder` > 8.1.3 requires NOT to set this up.

File: dist/bitbucket/RoboFile.php
Comment (fjgarlin) [inline dist/bitbucket/RoboFile.php:186]: Not sure why but localhost wasn't the default, so forcing it.

File: dist/bitbucket/RoboFile.php
Comment (fjgarlin) [inline dist/bitbucket/RoboFile.php:222]: Cypress released a major new version in May 2022 which breaks compatibility, so fixing version.

File: dist/circleci/.circleci/RoboFile.php
Comment (fjgarlin) [inline dist/circleci/.circleci/RoboFile.php:151]: Needed as otherwise we get permission denied.

File: dist/travisci/.travis.yml
Comment (fjgarlin) [inline dist/travisci/.travis.yml:2]: sqlite version for php 7.4 is very old and xenial didn't play well updating it.

File: dist/travisci/.travis.yml
Comment (fjgarlin) [inline dist/travisci/.travis.yml:29]: sqlite version for php 7.4 is very old and Drupal 9 requires a newer one.

File: dist/travisci/.travis/RoboFile.php
Comment (fjgarlin) [inline dist/travisci/.travis/RoboFile.php:238]: getting segmentation fault and nesting level limits reached with xdebug. not requiring it to run the test.

File: setup-bitbucket.sh
Comment (fjgarlin) [inline setup-bitbucket.sh:31]: Needs to be explicit to set it up properly.

**Comment by fjgarlin** [comment]
Travis and Circle down and fully tested. I might create some follow up issues, but this PR is to try to get it to work with latest versions and the images provided.

Will get to the other three platforms within the next days and it'll hopefully be quicker/easier. I'll also squash the commit to avoid a history of tests and switching on/off things.

File: dist/travisci/.travis.yml
Comment (fjgarlin) [reply dist/travisci/.travis.yml]: Changed the approach to run things inside the docker container so no longer needed.

**Comment by fjgarlin** [comment]
@juampynr - I tested all the changes on all five platforms and it's all good. I'll wait until tonight to merge, but not many changes since you approved.

**Comment by juampynr** [comment]
> @juampynr - I tested all the changes on all five platforms and it's all good. I'll wait until tonight to merge, but not many changes since you approved.

Just saw them. Everything looks fine!

**Pull request merged**
